client/coral-embed-stream/src/Comment.js

```diff
@@ -124,9 +123,16 @@ class Comment extends React.Component {
       commentIsIgnored,
     } = this.props;
 
-    const like = getActionSummary('LikeActionSummary', comment);
-    const flag = getActionSummary('FlagActionSummary', comment);
-    const dontagree = getActionSummary('DontAgreeActionSummary', comment);
+    const likeSummary = getActionSummary('LikeActionSummary', comment);
+    const flagSummary = getActionSummary('FlagActionSummary', comment);
+    const dontagreeSummary = getActionSummary('DontAgreeActionSummary', comment);
```

Comment:
I don't agree that dontagree is not in camel case :-D But that's a very minor thing ^^